src/test/run-pass/issue26709.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+struct Wrapper<T: ?Sized>(T);
+
+impl<T: ?Sized> Drop for Wrapper<T> {
+    fn drop(&mut self) {}
```

Comment:
I went for a check that uses a pointer stored in the struct, to make sure that `drop()` is called with the right arguments.